provider/resource_keycloak_openid_client.go

```diff
@@ -401,9 +406,23 @@ func resourceKeycloakOpenidClientCreate(data *schema.ResourceData, meta interfac
 		return err
 	}
 
-	err = keycloakClient.NewOpenidClient(client)
-	if err != nil {
-		return err
+	if data.Get("import").(bool) {
+		existingClient, err := keycloakClient.GetOpenidClientByClientId(client.RealmId, client.ClientId)
+		if err != nil {
+			return err
+		}
+		rootUrlString := data.Get("root_url").(string)
+		client.Id = existingClient.Id
+		client.RootUrl = &rootUrlString
```

Comment:
why are we only setting `RootUrl` for the client as opposed to other fields.  should we just call `getOpenidClientFromData` here?